libs/flowbite-angular/src/lib/components/dark-theme-toggle/dark-theme-toggle.component.ts

```diff
@@ -18,11 +18,11 @@ import { DomSanitizer } from '@angular/platform-browser';
   selector: 'flowbite-dark-theme-toggle',
   template: `
     <flowbite-icon
-      svgIcon="flowbite-angular:moon"
-      class="h-5 w-5 hidden dark:block" />
-    <flowbite-icon
-      svgIcon="flowbite-angular:sun"
+      svgIcon="solid:moon"
```

Comment:
Keep using `flowbite-angular:moon` and `flowbite-angular:sun`
It will crash on nes project as solid:moon and solid:sun are not registered